pkg/analyze/rook_analyze.go

```diff
@@ -0,0 +1,124 @@
+package analyzer
+
+import (
+	"encoding/json"
+	"strings"
+
+	"github.com/pkg/errors"
+	troubleshootv1beta1 "github.com/replicatedhq/troubleshoot/pkg/apis/troubleshoot/v1beta1"
+)
+
+const (
+	CephStatusGood string = "HEALTH_OK"
+	CephStatusWarn string = "HEALTH_WARN"
+	CephStatusErr  string = "HEALTH_ERR"
+)
+
+type CephHealth struct {
+	Status string `json:"status"`
+}
+
+type CephStatus struct {
+	Health CephHealth `json:"health"`
+}
+
+func analyzeRook(analyzer *troubleshootv1beta1.RookStatus, getRookCollection func(string) ([]byte, error)) (*AnalyzeResult, error) {
+	title := analyzer.CheckName
+	if title == "" {
+		title = "Rook Status"
+	}
+
+	result := &AnalyzeResult{
+		Title:   title,
+		IconKey: "", // TODO: we'll eventually remove the IconKey
```

Comment:
what is it?  if it's empty always , can we just remove it from this struct?  isn't "" the default?

Reply:
Grayson mentioned we'll eventually remove it.  I left a comment to be explicitly clear, so that nobody thinks this might be a bug or something.  I can remove it if you want.

Reply:
I removed it.